### PR TITLE
Add macOS toolchains for 10.12 Deployment Target

### PR DESCRIPTION
**Pull request by remyjette**

### CODE REVIEW

File: bin/detail/toolchain_table.py [hunk 1]
```diff
@@ -614,6 +614,9 @@ def verify(self):
       Toolchain('osx-10-13-dep-10-10', 'Xcode', osx_version='10.13'),
       Toolchain('osx-10-13-dep-10-10-cxx14', 'Xcode', osx_version='10.13'),
       Toolchain('osx-10-13-dep-10-10-cxx17', 'Xcode', osx_version='10.13'),
+      Toolchain('osx-10-13-dep-10-12', 'Xcode', osx_version='10.13'),
```
Comment (ruslo) [inline bin/detail/toolchain_table.py]: It's missing

File: bin/detail/toolchain_table.py [hunk 2]
```diff
@@ -622,6 +625,9 @@ def verify(self):
       Toolchain('osx-10-14-dep-10-10', 'Xcode', osx_version='10.14'),
       Toolchain('osx-10-14-dep-10-10-cxx14', 'Xcode', osx_version='10.14'),
       Toolchain('osx-10-14-dep-10-10-cxx17', 'Xcode', osx_version='10.14'),
+      Toolchain('osx-10-14-dep-10-12', 'Xcode', osx_version='10.14'),
```
Comment (ruslo) [inline bin/detail/toolchain_table.py]: Missing too

File: bin/detail/toolchain_table.py (hunk 1 above)
Comment (remyjette) [reply bin/detail/toolchain_table.py]: Whoops, thanks!

File: bin/detail/toolchain_table.py (hunk 2 above)
Comment (remyjette) [reply bin/detail/toolchain_table.py]: fixed, thanks!

**Pull request merged**